lib/dfda.ts

```
@@ -214,45 +236,46 @@ export async function searchGlobalVariables(
+  name?: string
+  type?: "global" | "user"
 }): Promise<GlobalVariable | UserVariable | undefined> {
-  const { id, name, type = 'user' } = params
+  const { id, name, type = "user" } = params
 
   try {
     // If ID is provided, use direct fetch methods
     if (id) {
-      const numericId = typeof id === 'string' ? parseInt(id) : id
+      const numericId = typeof id === "string" ? parseInt(id) : id
       if (!isNaN(numericId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)